internal/api/ui/login/custom_action.go

```diff
@@ -15,8 +16,16 @@ import (
 	iam_model "github.com/zitadel/zitadel/internal/iam/model"
 )
 
-func (l *Login) customExternalUserMapping(ctx context.Context, user *domain.ExternalUser, tokens *oidc.Tokens, req *domain.AuthRequest, config *iam_model.IDPConfigView) (*domain.ExternalUser, error) {
-	resourceOwner := req.RequestedOrgID
+func (l *Login) runPostExternalAuthenticationActions(
+	ctx context.Context,
+	user *domain.ExternalUser,
+	tokens *oidc.Tokens,
+	authReq *domain.AuthRequest,
+	httpReq *http.Request,
```

Comment:
why don't we just pass the request and use the context from there?
same for the other functions